examples/rag/finetune.py

```diff
@@ -328,12 +344,12 @@ def train_dataloader(self) -> DataLoader:
             // self.hparams.accumulate_grad_batches
             * float(self.hparams.max_epochs)
         )
-        scheduler = get_linear_schedule_with_warmup(
-            self.opt, num_warmup_steps=self.hparams.warmup_steps, num_training_steps=t_total
-        )
-        if max(scheduler.get_last_lr()) > 0:
-            warnings.warn("All learning rates are 0")
-        self.lr_scheduler = scheduler
+        # scheduler = get_linear_schedule_with_warmup(
```

Comment:
Oh right we can just comment out the scheduler if we don't care about accuracy. But we should still raise an issue for this right? This is a bug in their master branch.

Reply:
Yeah, that is a good idea (though wasn't a high priority for me yesterday)

Reply:
Also, i think the scheduler is actually already set in `configure_optimizer` in lightningbase